app/models/user.rb

```diff
@@ -525,6 +513,21 @@ def hotness_score
     search_score
   end
 
+  def connected_authentication_providers
```

Comment:
We have `Authentication::Providers.enabled_for_user(user)`, I think we should use that instead of adding a new method to `User`.